docs/index.md

```diff
@@ -12,27 +12,41 @@ hide:
 
 <!-- markdown-link-check-enable -->
 
+!!! info ""
+
+    Built for Low Power Wide Area Network Connectivity to Azure IoT Hub
+
 The LoRaWAN starter kit is an OSS cross platform private network implementation
 of the [LoRaWAN specification](https://lora-alliance.org/resource_hub/lorawan-specification-v1-0-2/)
 built for connectivity to Azure IoT Hub. It enables users to setup their own
 LoRaWAN network that can connect to LoRa based nodes (sensors) and send decoded
 message packets to Azure IoT Hub for cloud based processing, analytics and other
-workloads. Alternatively, it allows sending commands from the cloud to the end
+workloads.
+
+Alternatively, it allows sending commands from the cloud to the end
 nodes. The goal of the the project is to provide guidance and a reference for
 Azure IoT Edge users to experiment with LoRaWAN technology.
 
-![Architecture](images/EdgeArchitecture.png)
+![Architecture](images/EdgeArchitecture-updated.png)
```

Comment:
nit: if you want to rename the file, I would use a version rather than "updated". git is managing the history, so you can also argue that you can overwrite it and the 2.0 version shows the right image and so does < 2.0.

Reply:
fair point!